A tiny typo fix, removed license outfits, added Deep River transport variant

ReimeiSky opened the pull request:
One commit for `shipyards and outfitters.txt`, one commit for `world-forge.txt`.
One to address https://github.com/Wrzlprnft/world-forge/issues/40, one because I have read that doing otherwise would just let you purchase a license if you want a ship that needed it, even if that license is restricted to missions or not available on the planet.

Note that this is my first pull request, so be gentle~

ReimeiSky committed:
Unnecessary quotations, Deep River Transport

ReimeiSky committed:
Typo fix, removed license outfits

Tiny typo in the intro, and I removed the license outfits because we now
have license jobs.

Bladewood committed:
Removed ending quotation mark in intro message

Amazinite commented:
I'd keep quotations around single word outfits and ships, just for the sake of consistency. And on a related note, either every paragraph in the intro should have quotes around them or none of them should, because the first two paragraphs are in quotes while the second two are not.

Amazinite commented:
"Thanks for testing to:" should be "Thanks for testing **too:"** as well. :)

Bladewood committed:
Backticks on every line and changed a word

Either all quotes or all backticks. Because of line 19, it had to be backticks. Also, I changed "to:" at the end to "too:".

Bladewood committed:
Quotes everywhere

Put quotes where I removed them, and put quotes where there weren't before.

Wrzlprnft commented:
The fact that the player enters his name right after the "to" is on purpose, but i guess this way it feels better somehow.
